notebooks/getting_started/part3_exploring_cohorts.ipynb

```diff
@@ -1,15 +1,5 @@
 {
```

Comment:
<p>3D visualization is where itkWidgets really shine, so I would definitely include an example of how one can configure the widget to show 3D rendering!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>I touched on this in the comment above, but we have a few options here:</p><ol><li>Use the screenshot of the 3D image instead so that it matches the initial visualization</li><li>Separate the two into their own viewers so there will be two different screenshots - one for 2D, one for 3D</li><li>Just stick to the 3D example and users can just toggle to the slice view if they choose</li></ol><p>I am open to ideas, whatever you think feels more intuitive!</p>